x-pack/plugin/eql/qa/correctness/src/javaRestTest/java/org/elasticsearch/xpack/eql/EsEQLCorrectnessIT.java

```diff
@@ -72,27 +64,7 @@ public static void init() throws IOException {
 
     @Before
     public void restoreDataFromGcsRepo() throws Exception {
-        if (client().performRequest(new Request("HEAD", "/" + CFG.getProperty("index_name"))).getStatusLine().getStatusCode() == 404) {
-            highLevelClient().snapshot()
-                .createRepository(
-                    new PutRepositoryRequest(CFG.getProperty("gcs_repo_name")).type("gcs")
-                        .settings(
-                            Settings.builder()
-                                .put("bucket", CFG.getProperty("gcs_bucket_name"))
-                                .put("base_path", CFG.getProperty("gcs_base_path"))
-                                .put("client", CFG.getProperty("gcs_client_name"))
-                                .build()
-                        ),
-                    RequestOptions.DEFAULT
-                );
-            highLevelClient().snapshot()
-                .restore(
-                    new RestoreSnapshotRequest(CFG.getProperty("gcs_repo_name"), CFG.getProperty("gcs_snapshot_name")).waitForCompletion(
-                        true
-                    ),
-                    RequestOptions.DEFAULT
-                );
-        }
+        EqlDataLoader.restoreSnapshot(client(), highLevelClient(), CFG);
```

Comment:
The highLevelClient has a reference to the rest client already (through `getLowLevelClient()`), I don't think you need to pass `client()` here.

Reply:
Thx, missed that.